man/journalctl.xml

```diff
@@ -28,7 +28,12 @@
       <arg choice="opt" rep="repeat">MATCHES</arg>
     </cmdsynopsis>
   </refsynopsisdiv>
+  <refsect1>
+    <title>Quickstart</title>
 
+    <para>To get the last day of logs from nginx.service</para>
+    <para>journalctl -u nginx.service --since today</para>
```

Comment:
There is already `Examples` section. If you really want, then please update/extend the section.

Reply:
Point of this commit is to attempt to put the most commonly used information at the start of the docs.
As is - I find it odd that its buried at the very bottom

Meet in the middle and reorder the examples to have usages in front of the explanation of matching?

Reply:
Sorry, but we don't do that. This is documentation, not a guide.

Reply:
@clintonsteiner No, as @YHNdnzj already explained. Sorry.

Reply:
Appreciate the time and thanks for your work on this project